### PR TITLE
Implementation of sentiment analysis API

### PR DESCRIPTION
**Pull request by ArielSalazar04**
In this pull request, I have added the sentiment analysis API to my message HTML page.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/MessagesServlet.java [hunk 1]
```diff
@@ -24,29 +38,56 @@
...
-
-    @Override
-    public void init(){
-        messages = new ArrayList<>();
-    }
+    private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+        String languageCode = request.getQueryString().split("=")[1];
```
Comment (mickrobk) [inline portfolio/src/main/java/com/google/sps/servlets/MessagesServlet.java]: Looking up this by position is pretty error prone. (Eg what if you add another parameter in?) Could you parse the querystring instead and look for "languageCode" in the result? (Something like https://stackoverflow.com/questions/13592236/parse-a-uri-string-into-name-value-collection)

**Review by mickrobk — CHANGES_REQUESTED** [review]
This looks great overall!

File: portfolio/src/main/java/com/google/sps/servlets/MessagesServlet.java (hunk 1 above)
Comment (ArielSalazar04) [reply portfolio/src/main/java/com/google/sps/servlets/MessagesServlet.java]: Yes, thank you for pointing that out. At second glance I did not realize that I can use my getParameter() function in doGet just like I use it in doPost.

File: portfolio/src/main/java/com/google/sps/servlets/MessagesServlet.java [hunk 2]
```diff
@@ -63,9 +83,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             Entity messageEntity = new Entity("Message");
             Date date = new Date(System.currentTimeMillis());
 
-            messageEntity.setProperty("content", content);
-            messageEntity.setProperty("timestamp", date);
-            datastore.put(messageEntity);
+            Document doc =
+            Document.newBuilder().setContent(content).setType(Document.Type.PLAIN_TEXT).build();
```
Comment (mickrobk) [inline portfolio/src/main/java/com/google/sps/servlets/MessagesServlet.java:87]: Could you reformat these lines?

**Review by mickrobk — APPROVED** [review]

**Pull request merged**